Gemfile

```diff
@@ -2,6 +2,9 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'websocket-eventmachine-server', require: false
+gem 'websocket-eventmachine-client', require: false
+
```

Comment:
Why is this here but not in gemspec? Does this mean, we actually don't need invoker to understand websocket to process the requests?